docs/developer/how-to.md

```diff
@@ -17,3 +17,77 @@ Open your browser at:
 - http://localhost:8080/swagger-ui/ for the old swagger 1.2 API (old swagger, endpoints will disappear from here (and become available in the previous link) as time progresses)
 
 Swagger json (for swagger 2.0 endpoints) is available under http://localhost:8080/api/swagger.json
+
+## How to run federation tests across two backends
+
+Requirements:
+
+* `helm` and `kubectl` available on your PATH
+* access to a kubernetes cluster via e.g. a `~/.kube/config` file.
+
+Using the latest code on the `develop` branch:
+
+First, find the latest CI-compiled code made available as docker images:
+
+```
+# Run all commands from the top directory of wire-server
+make latest-tag
+```
+
+Output might be
+
+```
+./hack/bin/find-latest-docker-tag.sh
+latest tag for brig:
+2.104.11
+latest tag for nginz:
+2.104.11
+```
+
+Let's assume the tags are the same(*) for both, then export an environment variable:
+
+```
+export DOCKER_TAG=2.104.11
+```
+
+Now run the following, which will create two full installations of wire-server on the kubernetes cluster you've configured to connect to, and should take ~10 minutes.
+
+```
+make kube-integration-setup
+```
+
+Next, you can choose to either run all integration tests, which includes running the multi-backend integration tests, and takes another ~10 minutes. Or you can instead choose to *only* run the multi-backend tests.
+
+Run all integration tests (takes ~10 minutes) with: `make kube-integration-tests` - these tests run entirely on kubernetes.
```

Comment:
When I tried this, it worked fine, but I got no output until the very end. Is this expected? If so, maybe add a note to that effect.

Reply:
Yes, it's expected, kind of, but a limitation of using the underlying `helm test`. Maybe we want to run integration tests differently at some point, for better control, but that's another topic. I added a comment here.